### PR TITLE
Use Null Coalesce Operator

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

File: lib/Doctrine/DBAL/Connections/MasterSlaveConnection.php [hunk 1]
```diff
@@ -121,7 +121,7 @@ public function __construct(array $params, Driver $driver, Configuration $config
             $params['slaves'][$slaveKey]['driver'] = $params['driver'];
         }
 
-        $this->keepSlave = isset($params['keepSlave']) ? (bool) $params['keepSlave'] : false;
+        $this->keepSlave = (bool) $params['keepSlave'] ?? false;
```
Comment (morozov) [inline lib/Doctrine/DBAL/Connections/MasterSlaveConnection.php]: This doesn't look right. The `(bool)` casting seems to has higher [precedence](http://php.net/manual/en/language.operators.precedence.php) than `??`:
```php

$params = [];
$keepSlave = (bool) $params['keepSlave'] ?? false;

// Notice: Undefined index: keepSlave
```

Comment (carusogabriel) [reply lib/Doctrine/DBAL/Connections/MasterSlaveConnection.php]: So `isset` should be keep it here?

Comment (stloyd) [reply lib/Doctrine/DBAL/Connections/MasterSlaveConnection.php]:
```
$keepSlave = (bool) ($params['keepSlave'] ?? false);

**Review by Majkl578 — APPROVED** [review]

**Comment by Ocramius** [comment]
🚢  thanks @carusogabriel!

**Pull request merged**